src/divider.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
+import ThemeManager from './styles/theme-manager';
+import styleUtils from './utils/styles';
+
+const propTypes = {
+  /**
+   * CSS class that will be added to the divider's root element
+   */
+  className: React.PropTypes.string,
+
+  /**
+   * If true, the divider will be indented 72px
+   */
+  inset: React.PropTypes.bool,
+
+  /**
+   * Override the inline-styles of the list divider's root element
+   */
+  style: React.PropTypes.object,
+};
+
+const contextTypes = {
+  muiTheme: React.PropTypes.object,
+};
+
+const childContextTypes = {
+  muiTheme: React.PropTypes.object,
+};
+
+const defaultProps = {
```

Comment:
This is not supported yet by react-docgen.
We can keep it like this. I have opened an issue https://github.com/reactjs/react-docgen/issues/45.